lib/less/tree/color.js

```diff
@@ -132,6 +132,28 @@ Color.prototype.toHSL = function () {
     }
     return { h: h * 360, s: s, l: l, a: a };
 };
+Color.prototype.toHCG = function () {
```

Comment:
For HCG planned modifiers (for example "purity"). But I thinking about names, that will not conflict with HSL/HSV/HWB.